R/cost_uplift.R

```diff
@@ -0,0 +1,78 @@
+#' Uplift costs
+#'
+#' @param data episode data
+#' @param uplift
+#'
+#' @return episode data with uplifted costs
+#' @export
+cost_uplift <- function(data) {
+  data <- data %>%
+    # attach a uplift scale as the last column
+    uplift_set() %>%
+    dplyr::mutate(
+      cost_total_net = cost_total_net * uplift,
+      cost_total_net_incdnas = cost_total_net_incdnas * uplift,
+      apr_cost = apr_cost * uplift,
+      may_cost = may_cost * uplift,
+      jun_cost = jun_cost * uplift,
+      jul_cost = jul_cost * uplift,
+      aug_cost = aug_cost * uplift,
+      sep_cost = sep_cost * uplift,
+      oct_cost = oct_cost * uplift,
+      nov_cost = nov_cost * uplift,
+      dec_cost = dec_cost * uplift,
+      jan_cost = jan_cost * uplift,
+      feb_cost = feb_cost * uplift,
+      mar_cost = mar_cost * uplift
+    ) %>%
+    # remove the last uplift column
+    dplyr::select(-uplift)
+  return(data)
+}
+
+#' set uplift scale
+#'
+#' @param data episode data
+#' @param latest_cost_year default financial year '2223'
+#'
+#' @return episode data with a uplift scale
+#' @export
+uplift_set <- function(data, latest_cost_year = "2223") {
```

Comment:
Could we have a funtion for `latest_cost_year` please. See `R/00-update_refs.R`
```suggestion
uplift_set <- function(data, latest_cost_year = latest_cost_year()) {
```
(We probably wouldn't even need it as an argument then.